engine/classes/Elgg/Database/Plugins.php

```diff
@@ -183,7 +183,7 @@ public function generateEntities() {
 				if (!$plugin->isEnabled()) {
 					$plugin->enable();
 					$plugin->deactivate();
-					$plugin->setPriority('last');
+					$plugin->setPriority('new');
```

Comment:
is `last` no longer working? if it can not be used anymore... should we remove support?

Reply:
Last doesn't iterate, it just sets to max priority, which happens to be the last plugin in the list. I don't think it ever worked properly